x/sync/network_client.go

```diff
@@ -312,7 +312,6 @@ func (c *networkClient) Shutdown() {
 	defer c.lock.Unlock()
 
 	// reset peers
-	// TODO danlaine: should we call [Disconnected] on each peer?
```

Comment:
I don't think this is necessary

Reply:
q: why does this function exist at all? It doesn't seem to cancel any outstanding requests or cleanup any state

Reply:
removed